pyfr/integrators/base.py

```diff
@@ -41,7 +41,7 @@ def __init__(self, backend, rallocs, mesh, initsoln, cfg):
         self.nacptchain = 0
 
         # Current and minimum time steps
-        self._dt = cfg.getfloat('solver-time-integrator', 'dt')
+        self._dt = self.__dt = cfg.getfloat('solver-time-integrator', 'dt')
```

Comment:
This `__dt` is messy and confusing.

Reply:
I changed the name to _dt_far, similar to _dt_near